bigbluebutton-html5/imports/ui/components/user-list/user-list-item/user-actions/component.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import Icon from '/imports/ui/components/icon/component';
+import styles from './styles.scss';
+
+const propTypes = {
+  user: React.PropTypes.shape({
+    id: React.PropTypes.string.isRequired,
+  }).isRequired,
+
+  currentUser: React.PropTypes.shape({
+    isModerator: React.PropTypes.bool.isRequired,
+  }).isRequired,
+};
+
+const defaultProps = {
+};
+
+class UserActions extends Component {
+  constructor(props) {
+    super(props);
+
+    this.handleOpenChat = this.props.userActions.openChat.bind(this);
+    this.handleClearStatus = this.props.userActions.clearStatus.bind(this);
+    this.handleSetPresenter = this.props.userActions.setPresenter.bind(this);
+    this.handlePromote = this.props.userActions.promote.bind(this);
+    this.handleKick = this.props.userActions.kick.bind(this);
+  }
+
+  render() {
+    const {
+      user,
+      currentUser,
+    } = this.props;
+
+    return (
+      <div key={user.id} className={styles.userItemActions}>
+        <ul className={styles.userActionsList}>
+          {this.renderOpenChatAction()}
+          {currentUser.isModerator ? this.renderClearStatusAction() : null}
```

Comment:
All those renders have the same html markup, can you reuse the same render method just changing parameters?